functions.php

```diff
@@ -45,3 +46,46 @@ function pullData(PDO $db):array{
 $players = $query->fetchAll();
 return $players;
 }
+
+
+
+
+/**
+ * This function prepares the stats for entry into the databse by binding them to params and making a connection to the db
+ *
+ * @param string $name
+ * @param int $points
+ * @param int $games
+ * @param int $rings
+ * @param PDO $db
+ */
+function insertData ($name,$points,$games,$rings,$db){
+    $dbInsert = $db->prepare("INSERT INTO `nba` (`name`, `points`, `games`, `rings`) VALUES (:player, :points, :games, :rings)");
+    if (strlen($name <= 30)){
```

Comment:
You have a bracket in the wrong place here - it should look like this:

`if (strlen($name()) <= 30)`